### ISSUE TITLE
Missing file comment in incoming SlackFile

### ISSUE THREAD

**Issue by aabiryukov** [noobot/SlackConnector#78]
Additional to issue #9

When SlackConnector deserializing file data it loses file comment. Slack response has file comment in json field: file.initial_comment.comment. 
It is major metadata for the file message.

Example of file message with comment:
![slack_file_comment](https://user-images.githubusercontent.com/6473357/43217548-9d2e60be-904a-11e8-8f87-6440856bb8a4.png)

As result, is not possible to get file comment from incoming SlackFile.

**Comment by aabiryukov** [comment]
Surpise from slack. 
Yesterday Slack changed the format of message with image. Now SlackConnecter has empty file info :(
News (without details): https://get.slack.help/hc/en-us/articles/115004846068-Slack-updates-and-changes#july-2018

New message format has multiple files:
```
{
. . .
  "text": "USER MESSAGE",
  "files": [
    {
. . .
    }
  ],
}
```

**Comment by Workshop2** [comment]
https://api.slack.com/changelog/2018-05-file-threads-soon-tread

**Comment by Workshop2** [comment]
`file.initial_comment.comment` isn't a valid property. You can find the message the file was uploaded with under the SlackMessage text

**Comment by Workshop2** [comment]
File -> Files released in `4.2.71`
